api_documentation.raml

```diff
@@ -0,0 +1,301 @@
+#%RAML 1.0
+---
+title: careerLattice API
+baseUri: http://localhost:8002
+
+documentation:
+  - title: Welcome
+    content: |
+      Welcome to the Career Lattice Documentation. 
+
+  - title: Chapter one
+    content: |
+      More content here. Including **bold** text!
+
+/roles:
+    description: Collection of jobs and all the capabilities and families they belong to
+    get:
+        description: Gets an array of JSON objects containing roleID, roleName, capabilityID, capabilityName, jobFamilyID and jobFamilyName from the DB
+        responses:
+            200:
```

Comment:
only documents the happy path, does not show how your APIs handle errors and what response codes it returns as a result